pkg/kubelet/events/event.go

```diff
@@ -34,6 +34,9 @@ const (
 	FailedToCreatePodContainer     = "FailedCreatePodContainer"
 	FailedToMakePodDataDirectories = "Failed"
 	NetworkNotReady                = "NetworkNotReady"
+	StartingVerticalScalingPod     = "StartingVerticalScalingPod"
+	FailedToVerticalScalingPod     = "FailedVerticalScalingPod"
+	FinishedVerticalScalingPod     = "FinishedVerticalScalingPod"
 )
```

Comment:
Can you please add two more events named BeginContainerResize and CompletedContainerResize, and use those values in updatePodContainerResources function?